src/main/java/kr/co/khacademy/semi/model/Product.java

```diff
@@ -27,13 +27,47 @@ public class Product {
     
 
     public static Product of(HttpServletRequest req) {
+        
+        if(validateProductTitle(req.getParameter("title"))) {
+            throw new IllegalArgumentException();
+        }
+        
+        if(validateProductSummary(req.getParameter("summary"))) {
+            throw new IllegalArgumentException();
+        }
+        
+        if (validateProductDetail(req.getParameter("detail"))) {
+            throw new IllegalArgumentException();
+        }
+        
         return Product.builder()
+            .name(req.getParameter("name"))
+            .title(req.getParameter("title"))
+            .summary(req.getParameter("summary"))
+            .detail(req.getParameter("detail"))
+            .price(Long.valueOf(req.getParameter("price")))
+            .quantity(Long.valueOf(req.getParameter("quantity")))
+            .categoryId(Long.valueOf(req.getParameter("category_id")))
             .build();
     }
     
-   public static Product of(ResultSet resultSet) {
-     
-       try {
+
+    
+    private static boolean validateProductTitle(String title) {   
+        return (title.length() >= 3) && (title.length() <= 200);
```

Comment:
```suggestion
        return (3 <= title.length()) && (title.length() <= 200);
```

위와 같은 방법은 어떤가요?? 좀더 범위를 잘 알수 있을것 같아요~